### PR TITLE
Test using Jest

### PR DESCRIPTION
**Pull request by Bishoy-Samwel**
- [x]   Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test.
- [x]  Mock a storage object to "imitate" localStorage operations.
- [x]  Mock HTML to test if add/delete functions add or remove exactly one li element to/from the list in the DOM.

### CODE REVIEW

File: coverage/lcov-report/block-navigation.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```
Comment (carlos-ssh) [inline coverage/lcov-report/block-navigation.js:1]: - Kindly enable your Linters.

File: coverage/lcov-report/block-navigation.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+var jumpToCode = (function init() {
+    // Classes of code we would like to highlight in the file view
+    var missingCoverageClasses = ['.cbranch-no', '.cstat-no', '.fstat-no'];
+
+    // Elements to highlight in the file listing view
+    var fileListingElements = ['td.pct.low'];
+
+    // We don't want to select elements that are direct descendants of another match
```
Comment (carlos-ssh) [inline coverage/lcov-report/block-navigation.js:9]: [OPTIONAL]
- Keep your project professional and remove commented lines.

File: coverage/lcov-report/base.css [hunk 3]
```diff
@@ -0,0 +1,272 @@
+/* eslint-disable */
```
Comment (carlos-ssh) [inline coverage/lcov-report/base.css]: - Remove commented lines.
- Fix linter issues.
Screen Shot 2021-07-29 at 04 59 24

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Bishoy-Samwel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- **Kindly enable your ESLint in your files. Are needed to run it**.
- Enable the correct [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for your Project.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove commented lines of code in your files. You can leave only rules for your Linters.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: coverage/lcov-report/block-navigation.js (hunk 1 above)
Comment (Bishoy-Samwel) [reply coverage/lcov-report/block-navigation.js:1]: Why I should enable linters for external code I didn't write?

Comment (Davidosky007) [reply coverage/lcov-report/block-navigation.js:1]: * Please note that for javascript projects the right linter to use is [Javascript linters](https://github.com/microverseinc/linters-config/tree/master/javascript) and not [HTML + CSS + JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) 
* The HTML+CSS+JS is made for the Portfolio project which needs Javascript for interactions as a result the linters was made for easy use

File: coverage/lcov-report/base.css (hunk 3 above)
Comment (Davidosky007) [reply coverage/lcov-report/base.css]: * done 👍

**Comment by Davidosky007** [comment]
# Hi @Bishoy-Samwel 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
done-gif


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**